R/object_usage_linter.R

```diff
@@ -261,3 +287,23 @@ get_imported_symbols <- function(xml) {
     )
   }))
 }
+
+known_used_symbols <- function(fun_assignment, interpret_extensions) {
+  unique(c(
+    if ("rlang" %in% interpret_extensions) extract_env_symbols(fun_assignment),
+    if ("glue" %in% interpret_extensions) extract_glued_symbols(fun_assignment)
+  ))
+}
+
+extract_env_symbols <- function(fun_assignment) {
+  env_names <- xml_find_all(
+    fun_assignment,
+    "
+    .//SYMBOL[text() = '.env']
+      /parent::expr
+      /parent::expr[OP-DOLLAR]
```

Comment:
Do you feel it's worth supporting LBB as well?

Reply:
https://github.com/search?q=lang%3AR+%2F%5B%5Ea-z0-9._%5D%5B.%5Denv%5C%5B%5C%5B%5B%22%27%5D%2F&type=code

Yea, there's enough hits there & I don't think it's much harder. Will do.